Atomic/syscontainers.py

```diff
@@ -286,6 +286,24 @@ def _gather_installed_files_info (checkout, name):
 
         return info, rpm_installed, installed_files_checksum
 
+    def _get_remote_location(self, remote_input):
```

Comment:
This function should handle two cases.  Either the user provided the path to the directory containing the `rootfs` directory (e.g. `/var/lib/atomic/containers/foo.0`) or the rootfs directory itself (e.g. `/var/lib/atomic/containers/foo.0/rootfs`.

In the previous version, `remote_rootfs` pointed to `/var/lib/atomic/containers/foo.0/rootfs` while `remote_path` pointed to `/var/lib/atomic/containers/foo.0`.

I think to minimize changes in the code, we could make this function return both values.

What do you think?

Also, the `_resolve_remote_path` function should probably be merged in this new function.

Reply:
```
This function should handle two cases. Either the user provided the path to the directory containing the rootfs directory (e.g. /var/lib/atomic/containers/foo.0) or the rootfs directory itself (e.g. /var/lib/atomic/containers/foo.0/rootfs.

In the previous version, remote_rootfs pointed to /var/lib/atomic/containers/foo.0/rootfs while remote_path pointed to /var/lib/atomic/containers/foo.0.
```
I agree, but for both cases remote path will point to `/var/lib/atomic/containers/foo.0` right?  That is why I kinda move toward the direction of making remote rootfs outside out of the function. Due to the reasoning explained in the other review comment. 

Tho that being said, I can definitely also go with the return route as well, let me know your thoughts on this, and I can change it =).

 >Also, the _resolve_remote_path function should probably be merged in this new function.

Yea, good point, I will do that in a fixup

Reply:
I was thinking more of keeping the possibility of having the `rootfs` somewhere else, we could analyze the `config.json` for example.  This is not done now but we could decide to do it at some point.

Anyway, since it is not done now, it doesn't matter for the current patch to think of this case.  I am fine either way.